### DISCUSSION TITLE
How to know which links/urls are vulnerable?

### DISCUSSION THREAD

**Discussion by harvardstudent2020** [projectdiscovery/nuclei#556] [Q&A] [answered]
The output file.txt is empty![image](https://user-images.githubusercontent.com/69164471/108520124-5218b300-72f0-11eb-89ed-0d339a3ae8f2.jpeg)

**Comment by geeknik** [comment, marked as answer]
An empty output file is usually the result of not finding anything vulnerable but without any more information it's hard to debug.

**Reply by harvardstudent2020** [reply]
thanks for your insightful comment. my query is why is it showing so many vulnerabilities then? doesn't make sense.

**Reply by geeknik** [reply]
The information you see displayed in your screenshot is nuclei loading the templates that you asked for via `-t` and `-severity`. If you'd rather not see that information on the screen, please use the `-silent` flag.

**Reply by harvardstudent2020** [reply]
ohhh I see now. thanks a ton mate

**Reply by geeknik** [reply]
You're welcome. Good luck out there!

**Reply by harvardstudent2020** [reply]
thanks a ton bro. take care
